### PR TITLE
Typos in chapter 6

### PR DESCRIPTION
**Pull request by Julian-Wollersberger**
This book is really nice! 
Here are some mistakes or incosistencies I noticed while reading.

I'm not shure I understand what this sentence means: "3. We register an interest in `Read` events on this socket with a second syscall to register our interest in the event queue we created in (1)." 
Could you rephrase it?

### CODE REVIEW

**Comment by cfsamson** [comment]
Thanks for the feedback and for the corrections. Hmm, that sentence was confusing. I reprhased it to: 

"3. Through another syscall we register an interest in `Read` events on this socket. It's important that we also inform the OS that we'll be expecting to receive a notification when the event is ready in the event queue we created in (1)."

**Comment by Julian-Wollersberger** [comment]
Now it makes sence. Thanks!
